stubs/flake8-bugbear/bugbear.pyi

```diff
@@ -0,0 +1,26 @@
+import argparse
+import ast
+from typing import Any
+
+class BugBearChecker:
+    name: str
+    version: str
+    tree: ast.AST | None
+    filename: str | None
+    lines: list[str] | None
+    max_line_length: int
+    visitor: Any
```

Comment:
`ast.NodeVisitor` maybe?

Reply:
Actually, `Type[ast.NodeVisitor]`, i.e. any class that inherits from `ast.NodeVisitor`.